ci/fireci/fireci/uploader.py

```diff
@@ -59,3 +64,13 @@ def _construct_request_endpoint():
 def _get_commit_hash(revision):
   result = subprocess.run(['git', 'rev-parse', revision], stdout=subprocess.PIPE, check=True)
   return result.stdout.decode('utf-8').strip()
+
+
```

Comment:
remove extra line

Reply:
If I remove one blank line, the IDE complaints with `PEP 8: E302 expected 2 blank lines, found 1`.

Seems like they are top level methods and [python style guide](http://go/pystyle#blank-lines) does require two blank lines between them.